SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp

```diff
@@ -54,10 +54,6 @@ DEFINE_LOG_CATEGORY(LogSpatialGDKSimulatedPlayerDeployment);
 
 namespace
 {
-	//Windows Platforms
-	const FString Win64(TEXT("Win64"));
```

Comment:
Keep the Win64 constant and pass it directly to `BuildAssembly` in `NextStep`.

Reply:
this constant lives in the SpatialGDKEditorPackageAssembly.cpp now.